pkgs/tools/admin/azure-cli/extensions-manual.nix

```diff
@@ -1,5 +1,7 @@
-{ mkAzExtension
-, python3Packages
+{
+  mkAzExtension,
+  mycli,
+  python3Packages,
 }:
```

Comment:
This is still formatted RFC style.

Reply:
Reformatted via `nixfmt-classic`. Btw is that the "official" formatter for `azure-cli` at the moment?

Because right now I get diffs for `nixfmt-rfc-style` (obviously) and `alejandra`, but also nixfmt-classic`.

Reply:
Previously to nixfmt-rfc-style, [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt) was/is the common formatter for nixpkgs. Sorry for the trouble.